src/__init__.py

```diff
@@ -160,24 +156,23 @@ def setup_mongodb(application):
     mongo_user = os.environ.get("MONGODB_USERNAME")
     mongo_pass = os.environ.get("MONGODB_PASSWORD")
     mongo_db = os.environ.get("MONGODB_DATABASE")
-    mongo_host = os.environ.get("MONGODB_HOST", "mongo")
+    mongo_host = os.environ.get("MONGODB_HOST")
```

Comment:
Why did you get rid of the default?

Reply:
because it's sort of a dumb default and I'd rather they configure it. I was using it temporarily to test in docker, but it's best to insist it's configured either through the URI or the other env vars, or else exit with an error